site/resources/js/enhancement.js

```diff
@@ -10,6 +10,62 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+// Rating
+(function() {
+  function setFocus(rating, focused) {
+    rating.classList[focused ? 'add' : 'remove']('is-focused');
+  }
+
+  function setValue(rating, value) {
+    var input = rating.querySelector('.spectrum-Rating-input');
+    input.value = value;
+    Array.prototype.slice.call(rating.querySelectorAll('.spectrum-Rating-icon')).forEach(function(el, index) {
```

Comment:
Does this have to be added to the Array prototype for the slice method?

Reply:
Here we're invoking the `slice` method of `Array` with the `this` value set to the `NodeList` returned by `querySelectorAll`. This is a common mechanism to convert a `NodeList` to an array.

We could invoke `forEach` directly here instead of converting to an Array first. I'll go ahead and make that change.

Reply:
you could use the spread operator instead:
``[...rating.querySelectorAll(".spectrum-Rating-icon")]`
https://stackoverflow.com/a/44439293

Reply:
@bernhard-adobe we can't use new features here, this is ES5 land (no babel involved).

Reply:
oh, in this case carry on.